src/api_mocks/aggregator_api.rs

```diff
@@ -35,19 +35,18 @@ pub fn mock() -> impl Handler {
                         vdafs: Default::default(),
                         query_types: Default::default(),
                         protocol: random(),
-                        features: if random() {
-                            Feature::TokenHash.into()
-                        } else {
-                            Default::default()
-                        },
+                        features: Features::from_iter([Feature::TokenHash]),
```

Comment:
we still support dap-04

Reply:
This was a fix so that I could get [this test to work](https://github.com/divviup/divviup-api/pull/809/files#diff-5aac62dafc0bea54f44613fb71b32e29025bab53fbb7d39ee48ac6dcc05115e6R1060).

The [aggregator test fixture](https://github.com/divviup/divviup-api/blob/0e4a5ab076fd1c182d2b5a048e005bd7876ffebd/test-support/src/fixtures.rs#L156) needs to match the [mocked aggregator API](https://github.com/divviup/divviup-api/blob/0e4a5ab076fd1c182d2b5a048e005bd7876ffebd/src/api_mocks/aggregator_api.rs#L30) for this test to work. We're getting away with some other consistencies between the two, but I'd prefer to address it in a different set of work than this one.

I also would like this to have both features or a random selection of features, but there's another issue with using `assert_same_json_representation` insofar as arrays with different ordering fail the assertion.